### ISSUE TITLE
Docs say browser is supported as mock

### ISSUE THREAD

**Issue by rbarriuso** [mibrito707/cordova-plugin-secure-storage-echo#58]
`README.md` says:

```
Browser

The browser platform is supported as a mock only. Key/values are stored unencrypted in localStorage.
```

Nevertheless it seems the mock browser support was removed after v2.6.8. I think this should be reflected in the documentation.

**Comment by ludufre** [comment]
I agree. For Browser, I use https://github.com/softvar/secure-ls it's a very good.

**Comment by rbarriuso** [comment]
Thanks for the advice @ludufre !

**Comment by silviuburceadev** [comment]
It's a shame that browsers are not supported anymore. If someone explains why it isn't working on browsers anymore, I'm willing to get my hands dirty and restore browser support.
